docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md

```diff
@@ -28,25 +23,29 @@ To create a graph space on the **Schema** page, follow these steps:
 2. On the **Graph Space List** page, click the **+ Create** button.
 3. On the **Create** page, do these settings:
 
-   a. **Name**: Specify a name to the new graph space. In this example, `basketballplayer` is used. The name must be distinct in the database.
+   a. **Name**: Specify a name to the new graph space. In this example, `basketballplayer` is used. The name must be distinct in the database. The name cannot be used keywords or reserved keywords as identifiers. For more information, see [keywords](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md).
+
+   b. **Vid type**: The data types of VIDs are restricted to `FIXED_STRING(<N>)` or `INT64`; a graph space can only select one VID type, `FIXED_STRING(32)` is used. For more information, see [VID](../../1.introduction/3.vid.md).
+
+   c. **Comment**: The remarks of a certain property or the space itself. The maximum length is 256 bytes. By default, there will be no comments on a space, `Statistics of basketball players` is used.
```

Comment:
Same issue. Why "there will be no comments on a space" then "`Statistics of basketball players` is used"? What is their connection?